assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -19,41 +19,87 @@
 .googlesitekit-overlay-notification {
 	background-color: $c-white;
 	border-radius: $br-sm;
-	bottom: 0;
-	box-shadow: 0 8px $grid-gap-desktop 0 rgba(0, 0, 0, 0.25);
-	max-width: 70vw;
-	position: fixed;
-	right: 0;
-	z-index: $z-index-sk-overlay-notification;
+	margin: $grid-gap-phone;
 
 	@media (min-width: $bp-tablet) {
+		margin: $grid-gap-desktop;
+	}
```

Comment:
I think we can wrap the component with `withWidgetComponentProps` to use the `Widget` here? However, I'm not sure if that's the most effecient way, since the margin is only relevant for the mobile view. Manually matching the margin seems like a solid choice here, but let me know what you think.

Reply:
Now that I take a closer look at it, I think this is fine. The margin looks a little extra in Storybook, but I see how it is essential in the plugin. Thanks!